src/i18n/en/nav.ts

```diff
@@ -18,190 +18,105 @@
  * `src/i18n/ja/nav.ts`: `'getting-started': 'はじめに',`
  */
 export default [
-	{ text: 'Start Here', header: true, type: 'learn', key: 'startHere' },
+	{ text: 'Welcome, world!', header: true, type: 'learn', key: 'welcomeWorld' },
 	{ text: 'Getting Started', slug: 'getting-started', key: 'getting-started' },
-	{ text: 'Installation', slug: 'install/auto', key: 'install' },
-	{ text: 'Editor Setup', slug: 'editor-setup', key: 'editor-setup' },
-	{ text: 'Upgrade Astro', slug: 'upgrade-astro', key: 'upgrade-astro' },
+	{ text: 'Installation and Setup', slug: 'install/auto', key: 'install/auto' },
+	{ text: 'Deploy your site', slug: 'guides/deploy', key: 'guides/deploy' },
+	{ text: 'Migrate to Astro', slug: 'guides/migrate-to-astro', key: 'guides/migrate-to-astro' },
+	{ text: 'Introductory Tutorial - Build a Blog', slug: 'tutorial/0-introduction', key: 'blog-tutorial' },
 
 	{ text: 'Core Concepts', header: true, type: 'learn', key: 'coreConcepts' },
-	{ text: 'Why Astro', slug: 'concepts/why-astro', key: 'concepts/why-astro' },
+	{ text: 'Why Astro?', slug: 'concepts/why-astro', key: 'concepts/why-astro' },
 	{ text: 'Astro Islands', slug: 'concepts/islands', key: 'concepts/islands' },
 
-	{ text: 'Tutorials', header: true, type: 'learn', key: 'tutorials' },
-	{ text: 'Build a Blog', slug: 'tutorial/0-introduction', key: 'blog-tutorial' },
-	{
-		text: 'Extend with Content Collections',
-		slug: 'tutorials/add-content-collections',
-		key: 'add-collections-tutorial',
-	},
-	{
-		text: 'Extend with View Transitions',
-		slug: 'tutorials/add-view-transitions',
-		key: 'add-transitions-tutorial',
-	},
-	// { text: 'Thinking with Islands', slug: 'tutorial/0-introduction', key: 'island-tutorial' },
-
-	{ text: 'Basics', header: true, type: 'learn', key: 'basics' },
-
-	{
-		text: 'Project Structure',
-		slug: 'basics/project-structure',
-		key: 'basics/project-structure',
-	},
-	{
-		text: 'Components',
-		slug: 'basics/astro-components',
-		key: 'basics/astro-components',
-	},
+	{ text: 'Learn the Basics', header: true, type: 'learn', key: 'basics' },
```

Comment:
```suggestion
	{ text: 'Learn the basics', header: true, type: 'learn', key: 'basics' },
```

Reply:
Our current standard is upper case for section headings in the nav bar, so I will leave all as that for now and then make one sweeping decision.